Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -675,7 +685,10 @@ class TIFFLoadingContext {
         }()));
 
         auto subifd_handler = [&](u32 ifd_offset) -> ErrorOr<void> {
-            m_next_ifd = ifd_offset;
+            if (auto result = set_next_ifd(ifd_offset); result.is_error()) {
+                dbgln_if(TIFF_DEBUG, "{}", result.error());
```

Comment:
Should this be a `dbgln()` instead of a `dbgln_if()`? Seems pretty broken if this happens.

Reply:
:magic_wand: